src/python/setup.py

```diff
@@ -104,7 +97,7 @@ def make_valid_pypi_version(raw):
     entry_points = {
         "console_scripts": scripts,
```

Comment:
do we also want to modify hashbang lines in scripts? I don't think they are actually used, so we can perphaps even remove them?